tests/behaviour/config/parameters.py

```diff
@@ -60,9 +60,13 @@ def parseWords(text):
 @parse.with_pattern(r"\d\d\d\d-\d\d-\d\d(?: \d\d:\d\d:\d\d)?")
 def parse_datetime(text: str) -> datetime:
     try:
-        return datetime.strptime(text, "%Y-%m-%d %H:%M:%S")
+        return datetime.strptime(text, "%Y-%m-%dT%H:%M:%S")
```

Comment:
Note: this won't ever run right? Because the regex above doesn't accept a `T` ever. Let's discuss why you made this change tomorrow

Reply:
This function was called also from inside the Python client, on top of it being called by Cucumber.